components/layout_2020/positioned.rs

```diff
@@ -445,33 +445,32 @@ impl HoistedAbsolutelyPositionedBox {
 
         let shared_fragment = self.fragment.borrow();
 
-        let inline_axis = solve_axis(
-            cbis,
-            pbm.padding_border_sums.inline,
-            pbm.margin.inline_start,
-            pbm.margin.inline_end,
-            /* avoid_negative_margin_start */ true,
-            &shared_fragment.box_offsets.inline,
-            size.inline,
-        );
-
-        let block_axis = solve_axis(
-            cbis,
-            pbm.padding_border_sums.block,
-            pbm.margin.block_start,
-            pbm.margin.block_end,
-            /* avoid_negative_margin_start */ false,
-            &shared_fragment.box_offsets.block,
-            size.block,
-        );
-
-        let margin = Sides {
-            inline_start: inline_axis.margin_start,
-            inline_end: inline_axis.margin_end,
-            block_start: block_axis.margin_start,
-            block_end: block_axis.margin_end,
+        let solve_inline_axis = |computed_size| {
+            solve_axis(
+                cbis,
+                pbm.padding_border_sums.inline,
+                pbm.margin.inline_start,
+                pbm.margin.inline_end,
+                /* avoid_negative_margin_start */ true,
+                &shared_fragment.box_offsets.inline,
+                computed_size,
+            )
+        };
+        let solve_block_axis = |computed_size| {
+            solve_axis(
+                cbis,
```

Comment:
We talked about this in the review, but this probably deserves some kind of comment since this code passes an inline size here when calculating the block axis.

@SimonSapin I think you originally wrote this code? Do you recall why this might be happening?

Reply:
Sorry, at this point I don’t recall this level of detail

Reply:
no worries, i added a todo in 64de48782dee31421cbbc3ceb5a521f83c1551bb